ScnSoft-Education/LeetCodeTraining/ReverseInteger/ReverseIntegerSolution.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Text;
+
+namespace LeetCodeTraining.ReverseInteger
+{
+    //https://leetcode.com/problems/reverse-integer/
+    public static class ReverseIntegerSolution
+    {
+        public static int Reverse(int x)
+        {
+            string positiveString = x.ToString();
+            bool isPositive = true;
```

Comment:
Hello @shomzascnsoft 
I fully agree that int is too much for sign, thank you for pointing this out, but I have a question now - what prevents us from using sbyte instead of bool? I believe it provides an ability for direct math calculations (e.g. (sbyte)sign * (int)positiveInteger) instead of branching/introducing conditions. Are there any pitfalls like casting issues or CLS compatibility? I guess readability shouldn't suffer as well because variable name "sign" is quite descriptive. I've tried to submit sbyte solutions as well:
![image](https://github.com/daniilnaumovich/ScnSoft-Education/assets/149603422/5f55222f-456b-4a55-a2e8-c5137b1d6c91)
I know there is a discussion about using of differrent than Int32 types, and I wonder whether in this specific case sbyte could be applicable, or are there any agreements/conventions which suggest bool instead of sbyte for number negativity storage. Thank you!

Reply:
I'm not aware about agreements/conventions about this, idea was to use 1 byte variable instead 4 bytes, both solutions with byte and bool is more less fine I guess.

Regarding LC performance accurateness, I'm not sure that it is always objective, results, diffeer from time to time launching the same code.
![image](https://github.com/daniilnaumovich/ScnSoft-Education/assets/101803780/71cc78db-42e7-4840-b1fb-d23cd192a3aa)
![image](https://github.com/daniilnaumovich/ScnSoft-Education/assets/101803780/8e0c77c0-4bda-427e-90c1-fac9e3c214c9)